geoPipe.py

```diff
@@ -341,41 +458,27 @@ def to_db(trail_df, region_code, tablename, schema=""):
 
 	return 1
 
+def add_osm_trails_within_polygon(polygon, region_code, park_name):
 
-def main():
-
-	sys.setrecursionlimit(10000)
-
-	MAX_REPAIR_DIST_METERS = 150
-	BUS_RADIUS_METERS = 800
-	LOOP_COMPLETION_THRESHOLD_METERS = 20
-
-	REGION = "Rhode Island" # Good for testing since its small
-	#REGION = "New York"
-	## specify country in cases where multiple same-named regions exist
-	COUNTRY = ""
-
-	""" Executes pipeline logic
-	Process:
-			1) queries OSM 
-			2) splits OSM elements by type (way or node), then creates dataframes for each
-			3) For each node in each way, replace the node ID with an dict object of: ID, lat, lon
-			4) Names each trail from tags
-	"""
-	print("Requesting trails for {}".format(REGION))
+	print("\nSleeping for " + str(SLEEP_BETWEEN_QUERIES_SEC) + " secs")
+	time.sleep(SLEEP_BETWEEN_QUERIES_SEC)
 
+	print("\nquerying ways from OSM")
 
-	# tqdm means "progress" in Arabic, this guy wraps iterables and predicts the time it'll take to run. 
```

Comment:
Wait did I write this, hahaha? Did you know it is also stands for "Te quiero demasiado" in Spanish

Reply:
No haha, this code was written by one of our interns from the University of Michigan.  They actually wrote this pipeline originally, I just updated the code a bit.